KizGetdown.html

```diff
@@ -58,10 +53,10 @@
               >
             </li>
             <li class="nav-item active px-3">
-              <a class="nav-link" href="KizGetdown.html">KizGetdown</a>
+              <a class="nav-link" href="kizGetdown.html">KizGetdown</a>
```

Comment:
on mac renaming files (camelcase difference only) is tricky. If you got to github, you'll notice the file is still `KizGetdown.html`. [Here's how to fix it](https://coderwall.com/p/mgi8ja/case-sensitive-git-in-mac-os-x-like-a-pro)